src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
Are we planning on keeping the old inheritance system for the long run or is that coming later? If we do stick with the current system, I think the simplified method declaration syntax is nice:

```js
vjs.Component.extend({
  init(player, options) {
    // ...
  }
})
```